### ISSUE TITLE
默认把不需要显示的底部显示出来了。拖动之后正常

### ISSUE THREAD

**Issue by yitong12580** [letsar/flutter_staggered_grid_view#147]
Container(
        width: MediaQuery.of(context).size.width,
        height: 180,
        child: StaggeredGridView.countBuilder(
          key: UniqueKey(),
          // physics: NeverScrollableScrollPhysics(),
          padding: EdgeInsets.only(left: 10, right: 10, top: 10),
          // scrollDirection: Axis.horizontal,
          
          crossAxisCount: 5,
          itemCount: 17,
          itemBuilder: (_, i) {
            return Container(
              width: 30,
              height: 30,
              color: Colors.red,
            );
          },
          staggeredTileBuilder: (int index) => new StaggeredTile.count(1, 1),
          mainAxisSpacing: 10.0,
          crossAxisSpacing: 4.0,
        ))

**Comment by letsar** [comment]
Can you leave meaningful issues, in plain english please?
